### PR TITLE
adds script to install tech preview1 serverless operator

### PR DESCRIPTION
**Pull request by invincibleJai**
- adds the script to install tech preview1 serverless operator
- updates imports 

https://docs.openshift.com/container-platform/4.1/serverless/installing-openshift-serverless.html

### CODE REVIEW

**Comment by invincibleJai** [comment]
@vikram-raj thanks vikram and yes that's how service-mesh works so i don't think script can do much here.

File: frontend/packages/knative-plugin/yamls/knative-serving/smmr.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+apiVersion: maistra.io/v1
+kind: ServiceMeshMemberRoll
+metadata:
+  name: default
```
Comment (sbose78) [inline frontend/packages/knative-plugin/yamls/knative-serving/smmr.yaml]: Should we create a special namespace and add that namespace to this member role? We typically discourage people from using `default` to deploy apps.

Comment (invincibleJai) [reply frontend/packages/knative-plugin/yamls/knative-serving/smmr.yaml]: makes sense @sbose78 any suggestions or shall i go with `mynamespace` as in docs https://docs.openshift.com/container-platform/4.1/serverless/installing-openshift-serverless.html#installing-service-mesh-member-roll_installing-openshift-serverless

**Comment by sbose78** [comment]
That looks good !

**Comment by christianvogt** [comment]
Thanks @invincibleJai for this script. It'll make it easier for the devs to setup serverless.

**Pull request merged**